src/core/main.c

```diff
@@ -1396,7 +1396,7 @@ static int become_shutdown(
 
         assert(pos < ELEMENTSOF(command_line));
 
-        if (streq(shutdown_verb, "reboot") &&
+        if ((streq(shutdown_verb, "reboot") || streq(shutdown_verb, "kexec")) &&
```

Comment:
Please use STR_IN_SET for this. Otherwise looks good.

Reply:
Sure thing, done - local compilation looks fine, I'll also retest on the HW to be sure